test/index.spec.js

```diff
@@ -0,0 +1,117 @@
+import { install, loop, Cmd, combineReducers } from '../src';
+import { createStore } from 'redux';
+
+describe('redux loop store enhancer', () => {
+  it('a looped action gets dispatched after the action that initiated it is reduced', done => {
```

Comment:
why are we doing these tests with a combined reducer? it seems like unnecessary additional complexity

Reply:
see above. just didn't want to rework all of these tests from scratch for now. that'll be another project. just got them running on jest for now.

Reply:
👍